fnet/network_test.go

```diff
@@ -255,6 +255,73 @@ func TestValidatePayloadSize(t *testing.T) {
 	}
 }
 
+func TestGenerateRandomPayload(t *testing.T) {
```

Comment:
you can add a test for sending all the 2 bytes from 0x0000 to 0xFFFF - but not sure the current echo handler can handle that raw binary as it expects I believe url-encoded / http standard - tbd